run/testing/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/GoogleCloudPlatform/golang-samples/run/testing
+
+go 1.15
+
+require (
+	github.com/GoogleCloudPlatform/golang-samples v0.0.0-20200828000024-04937a2b81cf
+	github.com/GoogleCloudPlatform/golang-samples/run/grpc-ping v0.0.0-20200828000024-04937a2b81cf
+	github.com/GoogleCloudPlatform/golang-samples/run/grpc-server-streaming v0.0.0-20200828000024-04937a2b81cf
```

Comment:
Please add `replace` directives to make sure these tests always use the local versions rather than this particular pseudo-version.

Reply:
Good catch, thanks.